pom.xml

```diff
@@ -15,6 +15,8 @@
 
     <!-- Maven plugin versions -->
     <maven.compiler.version>3.13.0</maven.compiler.version>
+    <maven.assembly.version>3.7.1</maven.assembly.version>
+    <maven.download.plugin.version>1.9.0</maven.download.plugin.version>
```

Comment:
nit: indentation

Reply:
I was puzzliing over whether the whitespace was right before I got what you mean :grin: